### ISSUE TITLE
using pytorch_generation.py: setting --temperature argument to any value causes a failure

### ISSUE THREAD

**Issue by tanselmi** [salesforce/ctrl#41]
Works just fine without defining a temperature argument.

error comes out as the following:

![pytorchtemperror](https://user-images.githubusercontent.com/6148194/66692168-5f844e80-ec61-11e9-9614-ea9ec218b97d.PNG)

**Comment by keskarnitish** [comment]
Should be fixed in https://github.com/salesforce/ctrl/commit/9a365edd0caa5b6e91cbbde4b72db1e117877e91

Owing to the argument order, PyTorch assumed probs when I meant them to be logits. 

Closing for now, reopen as necessary.

**Comment by tanselmi** [comment]
will check it out, thanks!
